docs/reference/troubleshoot.md

```diff
@@ -80,13 +80,43 @@ namespace. To get the current Crossplane logs, run the following:
 kubectl -n crossplane-system logs -lapp=crossplane
 ```
 
-Remember that much of Crossplane's functionality is provided by providers. You
-can use `kubectl logs` to view provider logs too.
-
 > Note that Crossplane emits few logs by default - events are typically the best
 > place to look for information about what Crossplane is doing. You may need to
-> restart Crossplane (or your provider) with the `--debug` flag if you can't 
-> find what you're looking for.
+> restart Crossplane with the `--debug` flag if you can't find what you're
+> looking for.
+
+## Provider Logs
+
+Remember that much of Crossplane's functionality is provided by providers. You
+can use `kubectl logs` to view provider logs too. By convention, they also emit
+few logs by default.
+
+```shell
+kubectl -n crossplane-system logs <name-of-provider-pod>
+```
+
+All providers maintained by the Crossplane community mirror Crossplane's support
+of the `--debug` flag. The easiest way to set flags on a provider is to create a
+`ControllerConfig` and reference it from the `Provider`:
+
+```yaml
+apiVersion: pkg.crossplane.io/v1alpha1
+kind: ControllerConfig
+metadata:
+  name: debug-config
+spec:
+  args:
+    - --debug
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws
+spec:
+  package: crossplane/provider-aws:v0.18.1
+  controllerConfigRef:
```

Comment:
can this controllerconfigRef be set after the Provider has already been created? e.g. provider is installed, something bad happens later on, then the user wants to add this debug flag.  Can they just create a ControllerConfig and set this reference, even though the Provider was already installed/created?